internal/policy/verify.go

```diff
@@ -184,20 +184,34 @@ func VerifyRelativeForRef(ctx context.Context, repo *git.Repository, initialPoli
 	// Verify each entry, looking for a fix when an invalid entry is encountered
 	var invalidEntry *rsl.ReferenceEntry
 	var verificationErr error
+
+	targetRefEntiresCount := 0
+
+	for _, entry := range entries {
```

Comment:
This check doesn't belong here, IMO. This would only check in the specified range, so there could well be additional entries for the tag not in the specified range. The duplication check for tags needs to happen separately.

Reply:
I have moved it into verifyTagEntry